### PR TITLE
Nikhil development

### PR DESCRIPTION
**Pull request by Nikhilbabu08**

### CODE REVIEW

File: src/components/nav-layout/NavBar.jsx [hunk 1]
```diff
@@ -0,0 +1,113 @@
...
+                            <img className="h-8 hidden sm:block" src="/vite.svg" alt="Company Logo" />
+                            <span className="ml-3 text-white font-semibold text-xl hidden sm:block">SERVICE CONNECT</span>
+                        </div>
+
+                        {/* Authenticated or Non-authenticated views */}
+                        <div className="flex items-center pr-2 sm:ml-6">
+                            {isAuthenticated ? (
+                                <>
+                                    {/* Notification Button */}
+                                    <button
```
Comment (JaseelaAlimahamed) [inline src/components/nav-layout/NavBar.jsx]: make this is an different component

File: src/App.jsx [hunk 2]
```diff
@@ -1,12 +1,15 @@
...
+import UserLayout from './layouts/UserLayout';
 // import'./App.css';
 function App() {
 
 
   return (
     <Routes>
-      <Route path="/*" element={<UserRoutes/>} />
+       <Route path="/*" element={<UserLayout />}>
+          <Route path="*" element={<UserRoutes />} />
```
Comment (JaseelaAlimahamed) [inline src/App.jsx]: why you added this?

**Review by JaseelaAlimahamed — APPROVED** [review]

File: src/App.jsx (hunk 2 above)
Comment (Nikhilbabu08) [reply src/App.jsx]: to keep the layout same for all pages, should i remove userlayout from here and restructure it another way ?

File: src/components/user/Navbar.jsx
Comment (JaseelaAlimahamed) [inline src/components/user/Navbar.jsx:1]: change the path of the file
src/components/user/layoutComponents/Navbar.jsx

**Review by JaseelaAlimahamed — APPROVED** [review]

**Pull request merged**